tests/ChangePassword.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(" Verify Admin able login ", async ({ page }) => {
+  await page.goto("http://localhost:3000/allLogs");
```

Comment:
seems like you're using localhost to navigate to the allLogs page, this should be development environment url . and also fetch this url from env file. otherwise it will break on dev and production environment.